fuzz/main.go

```diff
@@ -1,3 +1,5 @@
+// Package is used in conjunction with
```

Comment:
This is incomplete. Also, a Go package should only be documented once. It's probably the best to document it here, not in fuzz_test.go.